### DISCUSSION TITLE
Jamovi bug in binary logistic regression save feature

### DISCUSSION THREAD

**Discussion by Mikeszk98** [community/community#41050] [Actions]
### Select Topic Area

Bug

### Body

I am using Jamovi 2.3.18.0 on a Windows 10 computer.

Recently, I computed a binary logistic regression with a single continuous variable.  
Jamovi has a save option that inserts three columns of data into the data worksheet; predicted probability, residual and Cook's Distance.

The predicted probability column is blank. Jamovi put in a column header and correctly identified the data type but there is no data filled into the column. 

I attached a csv file for your review . See the column Jamovi Predicted, Column J.
CSV files may be imported to or exported from Jamovi.

I emailed Jamovi about the problem and they suggested that the github community may have a solution.

Thank you.
Mike Szymczuk

[Jamovi sample binary logistic regression w residuals.csv](https://github.com/community/community/files/10181304/Jamovi.sample.binary.logistic.regression.w.residuals.csv)

**Comment by jfullstackdev** [comment]
is this intentional to be posted here on GitHub Community ? seems you are talking with the other team members of that development

**Reply by Mikeszk98** [reply]
It was suggested by a Jamovi that I email Github. If I placed my issue with the wrong group, then please suggest the correct group.
Thanks.
